Loop/Managers/Alerts/UserNotificationAlertPresenter.swift

```diff
@@ -44,3 +46,86 @@ class UserNotificationAlertPresenter: AlertPresenter {
         }
     }
 }
+
+extension UserNotificationAlertPresenter: AlertManagerResponder {
+    func acknowledgeAlert(identifier: Alert.Identifier) {
+        DispatchQueue.main.async {
+            self.log.debug("Removing notification %@ from delivered notifications", identifier.value)
+            self.userNotificationCenter.removeDeliveredNotifications(withIdentifiers: [identifier.value])
+        }
+    }
+}
+
+fileprivate extension Alert {
+
+    enum Error: String, Swift.Error {
+        case noBackgroundContent
+    }
+
+    func getUserNotificationContent(timestamp: Date) throws -> UNNotificationContent {
+        guard let content = backgroundContent else {
+            throw Error.noBackgroundContent
+        }
+        let userNotificationContent = UNMutableNotificationContent()
+        userNotificationContent.title = content.title
+        userNotificationContent.body = content.body
+        userNotificationContent.sound = getUserNotificationSound()
+        // TODO: Once we have a final design and approval for custom UserNotification buttons, we'll need to set categoryIdentifier
+//        userNotificationContent.categoryIdentifier = LoopNotificationCategory.alert.rawValue
+        userNotificationContent.threadIdentifier = identifier.value // Used to match categoryIdentifier, but I /think/ we want multiple threads for multiple alert types, no?
+        userNotificationContent.userInfo = [
+            LoopNotificationUserInfoKey.managerIDForAlert.rawValue: identifier.managerIdentifier,
+            LoopNotificationUserInfoKey.alertTypeID.rawValue: identifier.alertIdentifier,
+        ]
+        let encodedAlert = try encodeToString()
+        userNotificationContent.userInfo[AlertUserNotificationUserInfoKey.alert.rawValue] = encodedAlert
+        userNotificationContent.userInfo[AlertUserNotificationUserInfoKey.alertTimestamp.rawValue] =
```

Comment:
I don't believe `userInfo[AlertUserNotificationUserInfoKey.alert.rawValue]` or `userInfo[AlertUserNotificationUserInfoKey.alert.rawValue]` are being used anywhere and can be removed, along with the alert string encoding methods.